### ISSUE TITLE
its blocked by norton

### ISSUE THREAD

**Issue by lilbinkyfr** [braycarlson/alphares#15]
i get 2.3 and it says cant run virus detected or smth like that

**Comment by braycarlson** [comment]
I am going to close this issue, because I have created a FAQ to address some of these common issues. You can read it here: https://github.com/braycarlson/alphares#faq. If you are still experiencing issues after reading the FAQ, then feel free to create a new issue.
